<launch>

  <node pkg="kinect_calibration" type="run_extrinsic_calib.py" name="kinect_ex_calib_first" output="screen">
  	<param name="cam_info_topic" value="/kinect_first/hd/camera_info"/>
  	<param name="image_topic" value="/kinect_first/hd/image_color"/>
  	<param name="frame_name" value="kinect_first_rgb_optical_frame"/>
  	<param name="object_frame_name" value="board1"/>
  	<param name="row_num" type="int" value="5"/>
  	<param name="column_num" type="int" value="7"/>
  	<param name="size" type="double" value="0.035"/>
  	<param name="target_type" type="int" value="0"/>
  </node>

  <node pkg="kinect_calibration" type="run_extrinsic_calib.py" name="kinect_ex_calib_second" output="screen">
  	<param name="cam_info_topic" value="/kinect_second/hd/camera_info"/>
  	<param name="image_topic" value="/kinect_second/hd/image_color"/>
  	<param name="frame_name" value="kinect_second_rgb_optical_frame"/>
  	<param name="object_frame_name" value="board2"/>
  	<param name="row_num" type="int" value="5"/>
  	<param name="column_num" type="int" value="7"/>
  	<param name="size" type="double" value="0.035"/>
  	<param name="target_type" type="int" value="0"/>
  </node>
  
</launch>
